z2/Cargo.toml

```diff
@@ -59,6 +59,7 @@ revm = {version = "14.0.3", features = ["optional_balance_check"]}
 rs-leveldb = "0.1.5"
 rustls = "0.23.16"
 serde = {version = "1.0.215", features = ["derive"]}
+scilla-parser = "2.0.0"
```

Comment:
Please bump the version of `scilla-parser` in `zilliqa/Cargo.toml` too so we don't have two versions lying around.

Reply:
thanks , done